src/prngs/chacha.c

```diff
@@ -0,0 +1,213 @@
+/* LibTomCrypt, modular cryptographic library -- Tom St Denis
```

Comment:
A review would be great. 

Should I create a separate issue like `Call for reviewers - PRNG/ChaCha20` ?